backend/database-collection-schemas/createAnnouncementCollection.py

```diff
@@ -0,0 +1,117 @@
+# SPDX-License-Identifier: MIT
+# SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+import os
+import json
+
+from appwrite.client import Client
+from appwrite.services.database import Database
+
+"""
+export APPWRITE_ENDPOINT=<http://localhost/v1>
+export APPWRITE_PROJECT=<project_id>
+export APPWRITE_API_KEY=<api_key>
+"""
+APPWRITE_ENDPOINT = os.environ.get("APPWRITE_ENDPOINT")
+APPWRITE_PROJECT = os.environ.get("APPWRITE_PROJECT")
+APPWRITE_API_KEY = os.environ.get("APPWRITE_API_KEY")
+
+
+client = Client()
+database = Database(client)
+
+(
+    client.set_endpoint(APPWRITE_ENDPOINT)
+    .set_project(APPWRITE_PROJECT)
+    .set_key(APPWRITE_API_KEY)
+)
+
+# # Cleanup all collections created before. Used only in development!
+# listCollection = database.list_collections()
+# for collection in listCollection["collections"]:
+#     result = database.delete_collection(collection_id=collection["$id"])
+
+
+createCollectionResult = database.create_collection(
+    "Announcements",  # Collection Name
+    ["*"],  # Read permissions
+    ["team:<team_id_goes_here>"],  # Write permissions
+    [
+        {
+            "label": "created_at",
+            "key": "created_at",
+            "type": "numeric",
+            "required": True,
+            "array": False,
+        },
+        {
+            "label": "updated_at",
+            "key": "updated_at",
+            "type": "numeric",
+            "required": False,
+            "array": False,
+        },
+        {
+            "label": "title",
+            "key": "title",
+            "type": "text",
+            "required": True,
+            "array": False,
+        },
+        {
+            "label": "content",
+            "key": "content",
+            "type": "text",
+            "required": True,
+            "array": False,
+        },
+    ],
+)
+print(createCollectionResult)
+
+# Create some fake data
+data = [
+    (1637100804, "Message _8_04", "Content for Message _8_04"),
+    (1637100704, "Message _7_04", "Content for Message _7_04"),
+    (1637100604, "Message _6_04", "Content for Message _6_04"),
+    (1637100504, "Message _5_04", "Content for Message _5_04"),
+    (1637100404, "Message _4_04", "Content for Message _4_04"),
+    (1637100304, "Message _3_04", "Content for Message _3_04"),
+    (1637100204, "Message _2_04", "Content for Message _2_04"),
+    (1637100104, "Message _1_04", "Content for Message _1_04"),
+]
+
+for d in data:
+    createDocumentResult = database.create_document(
```

Comment:
the script has line 5 lines of logic/function calling, the rest are data. So why do we need to separate the data from the code? This is much more obvious  the usage of data and code.

Reply:
Yes, you name the reason. It's few code mixed into large portions of data. It would be easy to put 5 lines into a function to get an overview over what is happening but it's difficult to get an overview over the code when they are distributed over 80 lines of code.

I noticed it when I wondered where the `createCollectionResult` came from.

Reply:
Not a fair point to me. If this function call has >=80 lines of data, than splitting is really needed. But this one has 33 l;ines of 4 different data arguments.

Reply:
Yes, it's basically okay. An extra function is not needed, I see. But it maybe could have improved readability by shifting both big data variables to the top of the file. But okay.